packages/cardpay-sdk/sdk/currency-utils/types.ts

```diff
@@ -0,0 +1,6 @@
+import BigNumber from 'bignumber.js';
+
+/**
+ * Arguments acceptable to a BigNumber constructor, currently equivalent to BigNumber.Value
+ */
+export type BigNumberish = BigNumber.Value;
```

Comment:
@topwebtek7 this was the only thing changed. `BigNumberish` was `string | number | BigNumber`. The definition of `BigNumber.Value`:

https://github.com/MikeMcl/bignumber.js/blob/ae9cecab22a105c063308dc96bdb37f1bffed976/bignumber.d.ts#L327

Reply:
Figured that since BigNumber has good types, we should be using them